docs/data/charts/lines/CSSCustomization.tsx

```diff
@@ -21,6 +21,8 @@ export default function CSSCustomization() {
           id: 'Years',
           dataKey: 'date',
           scaleType: 'time',
+          min: new Date(1990, 0, 1),
+          max: new Date(2018, 0, 1),
```

Comment:
Why was this necessary? 🤔

Reply:
On mobile D3 uses one tick every 10 years instaed of every 2 or 4 years. So the end date is 2020 leving 2 years without data at the end. For me it looks better with the area finishing at the end of the axis

![image](https://github.com/user-attachments/assets/8055f3e3-9a06-4f5d-b465-390fe5e87bbf)